getting-started/_subsections/re-upload-a-recording-to-archives.md

```diff
@@ -0,0 +1,47 @@
+## [Re-Upload a Recording to Archives](#re-upload-a-recording-to-archives)
+After download an archived recording from Cryostat it can be re-uploaded into
```

Comment:
I think this should read: "After downloading an archived recording from Cryostat, it can be re-uploaded into..."